pom.xml

```diff
@@ -116,13 +114,27 @@
             <url>http://repository.springsource.com/maven/bundles/external</url>
         </repository>
     </repositories>
+
     <pluginRepositories>
         <pluginRepository>
             <id>repo.jenkins-ci.org</id>
             <url>https://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+
     <build>
+        <resources>
+            <resource>
+                <directory>${project.basedir}</directory>
+                <includes>
+                    <include>ThirdPartyNotices.txt</include>
```

Comment:
I remember this may break mvn release as mvn release will generate some temp files under repo root that will be checked by checkstyle. Could you double check?

Reply:
Added the `resourceIncludes` configuration as discussed.